### DISCUSSION TITLE
Compilation of icub-main on the PC104

### DISCUSSION THREAD

**Discussion by pedrombmachado** [robotology/community#197] [General]
Dear all,
 I am compiling the icub-main, following the steps described on the [wiki](http://wiki.icub.org/wiki/Compilation_on_the_pc104) on the pc104 and I found an issue that looks like a typo:
```sh
cd $ICUB_DIR
rm CMakeCache.txt
ccmake $ICUB_ROOT/main
```
shouldn't it be:
```sh
ccmake $ICUB_ROOT
```

**Comment by pattacini** [comment]
Yes, definitely, thanks for pointing this out.
Fixed now.

**Comment by pedrombmachado** [comment]
Hi @pattacini ,
  You are always welcome.
